Competition2017/src/competition/subsystems/climbing/commands/IntakeAtLowPowerForTimeCommand.java

```diff
@@ -0,0 +1,61 @@
+package competition.subsystems.climbing.commands;
+
+import com.google.inject.Inject;
+
+import competition.subsystems.climbing.ClimbingSubsystem;
+import edu.wpi.first.wpilibj.Timer;
+import xbot.common.properties.DoubleProperty;
+import xbot.common.properties.XPropertyManager;
+
+public class IntakeAtLowPowerForTimeCommand extends BaseClimbingCommand {
```

Comment:
I would rename this to `AscendAtLowPowerForTimeCommand()`, as "intake" is used by the collector (or maybe belts).